apps/codeforafrica/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,36 @@
+import { Section, RichTypography } from "@commons-ui/core";
+import React from "react";
+
+import TwoToneBackground from "../TwoToneBackground";
+
+const PageHeader = React.forwardRef(function PageHeader(props, ref) {
+  const { title, subtitle } = props;
+  if (!title && !subtitle) {
+    return null;
+  }
+
+  return (
```

Comment:
```suggestion

  if (!title && !subtitle) {
    return null;
  }
  return (
```
returns should be together.